### DISCUSSION TITLE
[Feature Request] Notification for New Status Updates

### DISCUSSION THREAD

**Discussion by cstayyab** [cstayyab/WALC#19] [Feature Requests]
**Feathub Feature Link:** https://feathub.com/cstayyab/WALC/+3
**Is your feature request related to a problem? Please describe.**
It would be really nice if we can get a notification every time someone uploads a status.

**Describe the solution you'd like**
Whenever a status update is received, a notification with that person's name should be displayed. 

**Describe alternatives you've considered**
There is none except to go to status area every time I wanna check if someone uploaded a status update or not.

**Additional context**
None

**To view all feature requests and to vote for your favorite ones visit https://feathub.com/cstayyab/WALC This will help in determining which feature needs to be implemented first**

**Comment by Darkkal44** [comment]
that idea is so amazing..
no other clients have a feature like that!
it would be amazing if the devs of this client start working on this amazing feature!
